.github/workflows/deploy.yaml

```diff
@@ -10,6 +10,7 @@ on:
   push:
     branches: ['integration/**', 'master', 'release/**', 'avado', 'debug-deploy/**']
     paths-ignore:
+      - '.processes/**/*'
```

Comment:
Should be `.processes/*`

Reply:
Processes can have subdirectories, so current version is more general